tests/main/apparmor-prompting-integration-tests/create_multiple_allow.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/sh
+
+# A test that replying with allow always allows multiple files which match the
+# path pattern to be created, but doesn't allow other file creation.
+
+TEST_DIR="$1"
+
+echo "Run the prompting client in scripted mode in the background"
+prompting-client.scripted \
+	--script="${TEST_DIR}/script.json" \
+	--grace-period=1 \
+	--var="BASE_PATH:${TEST_DIR}" | tee "${TEST_DIR}/result" &
+
+sleep 1 # give the client a chance to start listening
+
+for name in test1.txt test2.md fail.txt test3.pdf ; do
+	echo "Attempt to write $name"
+	snap run --shell prompting-client.scripted -c "echo $name is written > ${TEST_DIR}/${name}"
+done
+
+sleep 5 # give the client a chance to write its result and exit
```

Comment:
Nitpick: If possible, I would try to get rid of all sleeps throughout this PR. Sleeps introduce unnecessary delay and mask the true wait condition. What about something like `timeout 5 bash -c 'while pgrep -f "prompting-client-scripted" > /dev/null; do true; done'`?

Reply:
I've replaced sleeps with timeouts like you suggested, and set the timeout duration as a variable in `task.yaml` which is passed into each test case script.

Thanks for the great suggestion about pgrep as well!